templates/modules/common/subscription-order-summary.hypr

```diff
@@ -0,0 +1,43 @@
+
+<table class="mz-ordersummary subscription-detail">
```

Comment:
Rename file name to `subscription-summary.hypr`

Reply:
renamed the file name.